Il2CppInterop.Generator/Passes/Pass61ImplementAwaiters.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Runtime.CompilerServices;
+using AsmResolver.DotNet;
+using AsmResolver.DotNet.Signatures;
+using AsmResolver.PE.DotNet.Cil;
+using AsmResolver.PE.DotNet.Metadata.Tables;
+using Il2CppInterop.Generator.Contexts;
+
+namespace Il2CppInterop.Generator.Passes;
+
+public static class Pass61ImplementAwaiters
+{
+    public static void DoPass(RewriteGlobalContext context)
+    {
+        var corlib = context.GetAssemblyByName("mscorlib");
```

Comment:
I feel like this should be a property on `RewriteGlobalContext`. If not, create one called `CorLib`.

Reply:
Sounds good, should I make it pass straight to indexing myAssemblies or cache an AssemblyRewriteContext in a field?

Reply:
Dictionaries have constant time lookup, so no need to cache.